### PR TITLE
added results page, made minor fixes

### PR DESCRIPTION
**Pull request by muskanbagrecha**

### CODE REVIEW

**Review by iHarryD — APPROVED** [review]
Code looks neat. Good to merge. Would recommend using max and min functions instead of setting max and min properties separately.

File: result.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        </nav>
+        <!-- Navbar ends -->
+
+        <!-- Main starts -->
+        <main class="main__container">
+            <h1 class="large-title">
+                Good job!! 🎉🎉
+            </h1>
+            <h3>Final Score:3/6</h3>
+            <section class="quiz__container">
```
Comment (PuneetP16) [inline result.html]: Suggestion: There could be space in here " Final Score: 3/6 "

File: css/login.css [hunk 2]
```diff
@@ -1,20 +1,14 @@
...
-    max-height: 44rem;
     min-height: fit-content;
     display: flex;
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    padding: 1rem;
+    padding: 2rem 3rem;
     box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
     margin: 4rem 0;
```
Comment (manojsarna) [inline css/login.css:13]: check for color vars

**Comment by shruthipatil11** [comment]
Code looks neat and clean.Looks good to merge

**Review by shruthipatil11 — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -84,8 +88,7 @@ <h2 class="medium-title">Featured Categories</h2>
                 <a href="quiz-types.html">
                     <div class="card card-vertical category__card">
                         <div class="card__img">
```
Comment (manojsarna) [inline index.html:98]: can use webp instead of jpg or png for

- Better Performance
- Compressed Size

**Review by j24m — APPROVED** [review]
As far as I know, you have followed all the good practices in your code, class names, consistent indentation. Coming to the UI, loved it, color choice is great. Good work :)

File: js/app.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+// Toggle theme
+function toggleTheme() {
+    let theme = document.querySelector('link');
+    console.log(theme);
```
Comment (manojsarna) [inline js/app.js]: use const

**Review by manojsarna — APPROVED** [review]
overall looks good!!

do check the comments

**Review by PuneetP16 — APPROVED** [review]
Your result page looks neat.  Looks Good to Merge.

Suggestion:The alignment of both questions seems off. 

image

**Pull request merged**